FoursquareTopTests/VenueDetailViewControllerDriver.swift

```diff
@@ -29,4 +31,63 @@ class VenueDetailViewControllerDriver {
             tester.waitForAbsenceOfViewWithAccessibilityLabel(categoryName)
         }
     }
-}
\ No newline at end of file
+    
+    func expectMenuButton(toBeVisible visible: Bool) {
+        if visible {
+            testCase.waitForViewWithLocalizedAccessibilityLabel(.VenueDetailSeeMenu)
+        } else {
+            testCase.waitForAbsenceOfViewWithLocalizedAccessibilityLabel(.VenueDetailSeeMenu)
+        }
+    }
+    
+    func expectAddressLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool) {
+        
+        expectView(withText: venue.address, withAccessibilityLabel: tr(.VenueDetailAddressAccessibilityLabel), toBeVisible: visible)
+        
+    }
+    
+    func expectStatusLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool) {
+        
+        expectView(withText: venue.status, withAccessibilityLabel: tr(.VenueDetailStatusAccessibilityLabel), toBeVisible: visible)
+      
+    }
+    
+    func expectRatingLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool) {
+        
+        expectView(withText: venue.formattedRating, withAccessibilityLabel: tr(.VenueDetailRatingAccessibilityLabel), toBeVisible: visible)
+        
+    }
+    
+    func expectPricingLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool)
+    {
+        expectView(withText: venue.formattedPrice, withAccessibilityLabel: tr(.VenueDetailPricingAccessibilityLabel), toBeVisible: visible)
+    }
+    
+    private func expectView(withText text: String? = nil, withAccessibilityLabel label: String? = nil, toBeVisible visible: Bool) {
+        
```

Comment:
if no text and no label are received, you should probably break. Consider adding a guard statement at the beginning of the method